ui/client/test/ExpressionSuggester-test.js

```diff
@@ -338,6 +338,12 @@ describe("expression suggester", () => {
       expect(suggestions).toEqual([{methodName: "toUpperCase", refClazz: {refClazzName: "java.lang.String"}}])
     }).then(done)
   })
+
+  it("should support safe navigation", (done) => {
+    suggestionsFor("#input?.barB.bazC?.", {row: 0, column: "#input?.barB.bazC?.".length }).then(suggestions => {
```

Comment:
You don't need to duplicate expression in second parameter anymore :)

Reply:
great, thanks :)